core/arch/opnd.h

```diff
@@ -915,6 +917,8 @@ extern const reg_id_t dr_reg_fixer[];
 #    endif
 /** Number of general registers */
 #    define DR_NUM_GPR_REGS (DR_REG_STOP_GPR - DR_REG_START_GPR + 1)
+/** Number of SIMD vector registers */
```

Comment:
nit: end punctuation; good to have "The" too perhaps, for more professional user-facing doxygen comment.

Hmm, I guess none of the ones below have end punctuation either: I would say go ahead and add a `.` to the new ones but maybe not worth touching the old in this CL.

Reply:
Done.